llvm/lib/Target/RISCV/RISCVInstrInfoVSDPatterns.td

```diff
@@ -1294,7 +1294,7 @@ defm : VPatWidenFPMulAccSDNode_VV_VF_RM<"PseudoVFWMACC",
                                         fpext_oneuse>;
 defm : VPatWidenFPMulAccSDNode_VV_VF_RM<"PseudoVFWMACCBF16",
                                         AllWidenableBFloatToFloatVectors,
-                                        riscv_fpextend_bf16_oneuse>;
+                                        fpext_oneuse>;
```

Comment:
I think you can remove the extop argument to VPatWidenFPMulAccSDNode_VV_VF_RM after this